airbyte-integrations/connectors/source-bing-ads/source_bing_ads/client.py

```diff
@@ -40,6 +40,8 @@ def __init__(
         customer_id: str,
         client_secret: str,
         client_id: str,
+        tenant_id: str,
+        redirect_uri: str,
```

Comment:
These parameters should be optional keyword arguments that default to `"common"` and `""`.